Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs

```diff
@@ -5,35 +5,141 @@
+        }
+
+        private bool IsUserInputValid(string userInput)
+        {
+            if (userInput == "1" || userInput == "2")
+            {
+                return true;
+            }
+
+            Console.WriteLine("Proszę wybrać numer z listy");
+            return false;
+        }
+
+        private void GoToUserChoice(string userChoice)
+        {
+            switch (userChoice)
             {
-                Console.WriteLine("To jest twoja notatka, która została zapisana do pliku txt: " + note);
-                SaveFile(note);
-                
+                case "1":
+                    WriteNewNote();
+                    break;
+                case "2":
+                    MenageUserPathInput();
+                    break;
+                default:
+                    Console.WriteLine("Błąd");
+                    break;
+            }
+        }
+
+        private void WriteNewNote()
+        {       
+            Console.WriteLine("Wprowadź notatkę (wciśnij 2 x <enter> aby zapisać):");
+            string note = Console.ReadLine();
+            ConsoleKeyInfo typedKey = Console.ReadKey();
+
+            if (typedKey.Key == ConsoleKey.Enter && SaveFile(note))
+            {                
+                Console.WriteLine("Zapisano.");                                
                 var mainMenu = new Notatnik_parser.MainMenu.MainMenu();
                 mainMenu.Run();
-            }            
-        }      
+            }
+            else if (typedKey.Key == ConsoleKey.Enter && !SaveFile(note))
+            {                
+                Console.WriteLine("Błędna ścieżka. Nie udało się zapisać.");                                
+            }
+        }
+
+        private bool SaveFile(string textNoteParameter)
+        {    
+            Console.WriteLine(pathInstructions, "zapisać do");
+            string pathAndFileName = Console.ReadLine();
+
+            try
+            {
+                using (FileStream fs = File.Create(pathAndFileName))
+                {
+                    Byte[] textNote = new UTF8Encoding(true).GetBytes(textNoteParameter);
+                    fs.Write(textNote, 0, textNote.Length);
+                }
+            }
+            catch (UnauthorizedAccessException)
+            {
+                FileAttributes attr = (new FileInfo(pathAndFileName)).Attributes;
+                return false;
+            }
+            
+            return true;
+        }
+
+        private void OpenTextFile(string insertPathValue)
+        {
+            using (FileStream fs = File.Open(insertPathValue, FileMode.Open)) 
+            {
+                byte[] b = new byte[1024];
+                UTF8Encoding temp = new UTF8Encoding(true);
+
+                while (fs.Read(b,0,b.Length) > 0) 
+                {
+                    Console.WriteLine(temp.GetString(b));
+                }
+            }
+        }
+
+        private bool IsUserPathInputValid(string insertPathValue)
```

Comment:
This method check input and open file. You should create a method to check input and open fil(this one you have :)) and use their separately in MenageUserPathInput